src/App.vue

```diff
@@ -80,6 +80,15 @@
                 <li>
                   <hr class="navbar-divider" />
                 </li>
+                <li>
+                  <router-link
+                    class="block px-4 py-2 hover:bg-blue-500 hover:text-white"
+                    :to="ownerProjectsLink('user', user.username)"
```

Comment:
I don't think `ownerProjectsLink` is the right link to use. Why not just `ownerLink`?

Reply:
> I don't think `ownerProjectsLink` is the right link to use. Why not just `ownerLink`?

Seems more correct 'ownerLink'. I'll replace it.